src/sfdxCommand.ts

```diff
@@ -68,10 +68,16 @@ export class Result implements SfdxResult {
  * Defines a varargs configuration. If set to true, there will be no
  * validation and varargs will not be required.  The validator function
  * should throw an error if validation fails.
+ *
+ * Should allow keys to be repeated. Values for repeated keys will either be
+ * presented in a concatenated string or as an array.
  */
 export type VarargsConfig =
   | {
       required: boolean;
+      allowMultipleKeys?: boolean;
```

Comment:
`allowMultipleKeys` will disable the duplicate key check. Defaults to false/undefined to protect backward compatibility.
`multipleValuesAsArray` defines how values for multiple keys will be presented. True, will place values into an array. False will cause all values to be presented in a single string with values separated by property `delimiter`. Defaults to false/undefined to protect backward compatibility.